pkg/kube/dockerconfig.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019-2022 the Kubeapps contributors.
```

Comment:
```suggestion
// Copyright 2023 the Kubeapps contributors.
```

Also, but unrelated, I think we should update the [NOTICE](https://github.com/vmware-tanzu/kubeapps/blob/main/NOTICE) file and remove this [old file](https://github.com/vmware-tanzu/kubeapps/blob/main/open_source_licenses.2.6.4.txt)

Reply:
Yep, I've been removing the old OSL license files later (I don't think we personally need to add them to the root of the repo anyway, given that we upload them to the release artifacts and leave them there.. wdyt?)

Reply:
> I don't think we personally need to add them to the root of the repo

The old ones? No, I don't think so. As for the current one (the one to be released), we need it just before tagging the release, this way it would end up in the release artifact.
Right after that... I believe we can safely remove it; in fact, that file no longer represents the current status of the project, as we might add/remove deps and, moreover, it is just a document to be embedded in a release artifact.

That said... I think it's easier for us to just remove it as part of the release process: remove the old one and add the new one.

Reply:
> Right after that... I believe we can safely remove it; in fact, that file no longer represents the current status of the project, as we might add/remove deps and, moreover, it is just a document to be embedded in a release artifact.

Which is why I think it's more useful uploading it as a (separate) release artifact for that release, rather than temporarily including it in the repo. But IANAL.